installscripts/Installer.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 #
 # File: rhel7Installer.sh
```

Comment:
@Feroz-Shaikh : Can you update the File name captured in the file to say Installer.sh?